app/controllers/sessions_controller.rb

```diff
@@ -6,8 +6,12 @@ def create
     user = User.find_by(email: session_params[:email])
     if user && user.authenticate(session_params[:password])
       session[:user_id] = user.id
-      flash[:success] = "#{user.first_name} Has Successfully Logged In"
-      redirect_to home_path
+      if current_user.roles.take.name == "admin"
+        redirect_to admin_dashboard_path
+      else
+        flash[:success] = "#{user.first_name} Has Successfully Logged In"
+        redirect_to home_path
+      end
```

Comment:
This is a lot! Any ideas on refactoring this logic @JonathanSR @AELSchauer @alfosco? Just something to remember for later.